packages/gluestick/src/commands/test/test.js

```diff
@@ -36,6 +36,7 @@ const getJestDefaultConfig = (aliases: Object, webpackRules: RegExp[]): string[]
   }
 
   const config = {
+    testRegex: '\\/__tests__\\/.*\\.(test|spec)\\.jsx?$',
```

Comment:
No, the default regex is `(/__tests__/.*|(\\.|/)(test|spec))\\.jsx?$` which will match any file that have `test` or `spec` as a name or every file in `__tests__` folder. This regex will mach files that are inside `__tests__` folder **and** have `test` or `spec` postfix.